packages/select/src/ListMenu.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback, useContext } from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { useViewportSize } from '@leafygreen-ui/hooks';
+import { keyMap } from '@leafygreen-ui/lib';
+import { breakpoints } from '@leafygreen-ui/tokens';
+import SelectContext from './SelectContext';
+import { colorSets, mobileSizeSet, sizeSets } from './styleSets';
+
+const menuStyle = css`
+  position: relative;
+  top: 5px;
+  width: 100%;
+  border-radius: 3px;
+  line-height: 16px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  overflow: auto scroll;
+`;
+
+interface ListMenuProps {
+  children: React.ReactNode;
+  id: string;
+  refElement: React.MutableRefObject<HTMLUListElement | null>;
+  onClose: () => void;
+  onSelectFocusedOption: () => void;
+  onFocusPreviousOption: () => void;
+  onFocusNextOption: () => void;
+}
+
+export default function ListMenu({
+  children,
+  id,
+  refElement,
+  onClose,
+  onFocusPreviousOption,
+  onFocusNextOption,
+  onSelectFocusedOption,
+}: ListMenuProps) {
+  const { mode, size } = useContext(SelectContext);
+
+  const colorSet = colorSets[mode];
+  const sizeSet = sizeSets[size];
+
+  const onKeyDown = useCallback(
+    (event: React.KeyboardEvent) => {
+      // No support for modifiers yet
+      /* istanbul ignore if */
```

Comment:
do we need these comments in production too?

Reply:
it only affects test coverage reporting. it should get stripped out when bundled